src/components/books.js

```diff
@@ -0,0 +1,15 @@
+import AddBooks from './addBook';
+import Book from './book';
+
+const Books = () => (
+  <div>
+    <Book title="The Hunger Games" author=" Suzanne Collins" />
+    <Book title="Dune" author="Frank Herbert" />
```

Comment:
Instead of repeating the Book component here, kindly consider using a method like the map (array.map) to loop through and list them. Please check [this link](https://reactjs.org/docs/lists-and-keys.html) for more explanation on how to do that. :pray: